src/runtime_src/core/tools/xbmgmt2/SubCmdExamine.cpp

```diff
@@ -121,6 +121,21 @@ SubCmdExamine::execute(const SubCmdOptions& _options) const
     return;
   }
 
+  // Determine report level
+  if (devices.size() == 0 && reportNames.size() == 0)
+    reportNames.push_back("host");
```

Comment:
warning: use emplace_back instead of push_back [modernize-use-emplace]

```cpp
note: this fix will not be applied because it overlaps with another fix
```